insights/parsers/multipath_conf.py

```diff
@@ -90,7 +100,7 @@ class MultipathConf(Parser, LegacyItemAccess):
         }
 
     Examples:
-        >>> conf = shared[MultipathConf]
+        >>> conf = shared[MultipathConfParser]
```

Comment:
Since the `MultipathConfParser` is a base class without any spec, which means the base class itself will never hit, I think better move the examples to the sub-class.